content/docs.md

```diff
@@ -49,69 +49,139 @@ later.
-merged when you are ready to publish the translations.
+<center><img alt="Screenshot of the Crowdin Settings showing the 'Hide (regular detection)' option." src="../images/duplicate_strings.png" width=800/></center>
 
-### Cleaning up
+{{< /admonition >}}
 
-After merging the translations PR, the Crowdin service branch (by default, named `l10n_main`) will have merge conflicts with `main`. To fix this, delete the Crowdin service branch. Crowdin will automatically recreate the service branch with merge conflicts resolved. This same process can also be used to resolve merge conflicts if translations are updated outside of Crowdin.
 
 ## Automation details
 
-Once a GitHub repository has been synced to Crowdin using the Scientific Python
-set up, the
-[`create_branch_for_language.sh` script](https://github.com/Scientific-Python-Translations/automations/blob/main/scripts/create_branch_for_language.sh)
-is used to create a new branch including all (and only) commits from Crowdin's
-branch for a particular language of interest. This script is useful because:
+### Cookiecutter Template
+
+Use the cookiecutter template to generate a translation-ready repo:
+
+```bash
+cookiecutter gh:Scientific-Python-Translations/translations-cookiecutter
+```
+
+When prompted, enter the details for your project (e.g. project name, organization, base branch). The structure will include necessary metadata, content folders, and pre-commit configuration. See the [Cookiecutter repository](https://github.com/Scientific-Python-Translations/translations-cookiecutter) for more information.
+
+### Sync Content from the Source Repo
+
+The `sync_content.yml` github workflow is in charge of keeping the **original source content** in sync with the translations repository within the Scientific Python Translations organization.
+
+This workflow uses the `content-sync` [Github action](https://github.com/Scientific-Python-Translations/content-sync). 
+
+### Example: Sync Scipy content
+
+```yaml
+name: Sync Content
+on:
+  schedule:
+    - cron: '0 5 * * *'
+  workflow_dispatch:
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync Scipy Content
+        uses: Scientific-Python-Translations/content-sync@main
+        with:
+          source-repo: "scipy/scipy.org"
+          source-folder: "scipy.org/content/en/"
+          source-ref: "main"
+          translations-repo: "Scientific-Python-Translations/scipy.org-translations"
+          translations-folder: "scipy.org-translations/content/en/"
+          translations-ref: "main"
+          # These are provided by the Scientific Python Project and allow
+          # automation with bots
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
+          token: ${{ secrets.TOKEN }}
+```
+
+This ensures your translation repo always has the latest source material in English. If the source content has changed since the latest workflow run, 
```

Comment:
```suggestion
This ensures your translation repo always has the latest source material in English (the chosen source language). If the source content has changed since the latest workflow run, 
```